devel/commitizen/Portfile

```diff
@@ -43,14 +43,17 @@ depends_run-append \
     port:py${python.version}-tomlkit \
     port:py${python.version}-yaml
 
-default_variants    +bash
+default_variants    +bash_completion
```

Comment:
I'll leave this up to @harens , but is changing the variant name really necessary?

Reply:
@reneeotten Firstly, most other packages use '+bash_completion'. Secondly, port 'bash-completion' is the real dependency we want, but not only 'bash'.

Reply:
Thank you very much for your changes @laggardkernel! This is a really useful contribution. ⭐ Also, thanks @reneeotten for your review.

I think changing the name is a good thing, mainly for standardisation. My only note would be to remove the variant description to fix the `port lint` warnings. Otherwise, LGTM. 👍

Reply:
that's sounds good to me. I would additionally suggest to change the commit message `'fix depends'` isn't that informative, perhaps just do `fix dependencies` instead? @harens I'll let you merge this once your satisfied with it.

Reply:
@harens Okay, fixed the warning and force pushed the change.

It seems 'bash_completion' variant is pre-defined by MacPorts, and we don't need to provide any variant description for it. And this fixes the warning.

Reply:
> It seems 'bash_completion' variant is pre-defined by MacPorts, and we don't need to provide any variant description for it. And this fixes the warning.

Yep, it's defined for you. In case you're interested, the full list of pre-defined descriptions can be found [here](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/variant_descriptions.conf).

> I would additionally suggest to change the commit message 'fix depends' isn't that informative, perhaps just do fix dependencies instead?

Just to get this merged, I made the change before commiting.

Thank you @laggardkernel for opening this PR! Your contribution is greatly appreciated.